news/henryiii-build-locally-python3.rst

```diff
@@ -0,0 +1,24 @@
+**Added:**
+
+* <news item>
+
+**Changed:**
+
+* `build-locally.py` now uses `python3` even if `python` is `python2` (Python 3.6+ was already required)
```

Comment:
This is rat, not me, so you need to use double backticks :(

Reply:
Ah, I see the file extension. Will correct tomorrow.

Reply:
Stupid autocorrect keyboard 😡